tests/acceptance/puede-ingresar-en-actividades-test.js

```diff
@@ -28,27 +29,40 @@ module('Acceptance | puede ingresar en actividades', function (hooks) {
           throw e;
         }
       }
-      const tituloVisibleEnPantalla = $(".challenge-title").text().trim();
-      assert.equal(tituloVisibleEnPantalla, tituloEsperado, "La actividad se llama " + tituloVisibleEnPantalla + " y no " + tituloEsperado);
+
+      //TODO: Make this work
+      var intl = Ember.inject.service();
```

Comment:
Si esto no te funciona creo que podés usar algo de este estilo:
```suggestion
      var intl = this.owner.lookup('service:intl')
```